### PR TITLE
Fix CMake style issues

### PR DESCRIPTION
**Pull request by IRDonch**
These were found with cmakelint.

### CODE REVIEW

File: demos/multi_channel_common/cpp/CMakeLists.txt [hunk 1]
```diff
@@ -17,13 +17,9 @@ if(NOT(OpenCV_FOUND))
     return()
 endif()
 
-file (GLOB MAIN_SRC
-        ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp
-        )
+file(GLOB MAIN_SRC ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp)
```
Comment (Wovchena) [inline demos/multi_channel_common/cpp/CMakeLists.txt:21]:
```suggestion
```

File: demos/multi_channel_face_detection_demo/cpp/CMakeLists.txt [hunk 2]
```diff
@@ -31,13 +31,9 @@ if(NOT(OpenCV_FOUND))
     return()
 endif()
 
-file (GLOB MAIN_SRC
-        ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp
-        )
+file(GLOB MAIN_SRC ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp)
```
Comment (Wovchena) [inline demos/multi_channel_face_detection_demo/cpp/CMakeLists.txt:35]:
```suggestion
```

File: demos/multi_channel_human_pose_estimation_demo/cpp/CMakeLists.txt [hunk 3]
```diff
@@ -31,13 +31,9 @@ if(NOT(OpenCV_FOUND))
     return()
 endif()
 
-file (GLOB MAIN_SRC
-        ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp
-        )
+file(GLOB MAIN_SRC ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp)
```
Comment (Wovchena) [inline demos/multi_channel_human_pose_estimation_demo/cpp/CMakeLists.txt:35]:
```suggestion
```

File: demos/multi_channel_object_detection_demo_yolov3/cpp/CMakeLists.txt [hunk 4]
```diff
@@ -31,13 +31,9 @@ if(NOT(OpenCV_FOUND))
     return()
 endif()
 
-file (GLOB MAIN_SRC
-        ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp
-        )
+file(GLOB MAIN_SRC ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp)
```
Comment (Wovchena) [inline demos/multi_channel_object_detection_demo_yolov3/cpp/CMakeLists.txt:35]:
```suggestion
```

**Review by Wovchena — CHANGES_REQUESTED** [review]

File: demos/multi_channel_common/cpp/CMakeLists.txt (hunk 1 above)
Comment (IRDonch) [reply demos/multi_channel_common/cpp/CMakeLists.txt]: All fixed.

**Review by Wovchena — APPROVED** [review]

**Pull request merged**